src/wasi/Wasi.cpp

```diff
@@ -108,12 +109,70 @@ WASI::WASI()
     init_options.argc = 0;
     init_options.argv = nullptr;
     init_options.envp = nullptr;
-    init_options.preopenc = 0;
+    if (shareHostEnv) {
+        std::vector<std::string> variables = {
+            // Common Unix environment variables.
+            "PATH",
+            "LIB",
+            "PWD",
+            "SHELL",
```

Comment:
I wonder about how did you define these common env variables
Is there any reference?

Reply:
Yes, I've used these sites:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08
https://learn.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables
https://pureinfotech.com/list-environment-variables-windows-10/

Please tell me if there are any more that I should add or if I should remove some